src/MissionManager/TransectStyleComplexItem.cc

```diff
@@ -1128,6 +1147,9 @@ void TransectStyleComplexItem::_buildAndAppendMissionItems(QList<MissionItem*>&
             } else {
                 _appendWaypoint(items, missionItemParent, seqNum, mavFrame, 0 /* holdTime */, coordInfo.coord);
             }
+            if (lastSurveyExit) {
```

Comment:
@DonLakeFlyer This variable is not working as intended. It is not entering this condition if there is a turnaround. It wasn't adding the camera mount command at the end of the survey mission.

Reply:
Are you sure you are fully up to date with latest master. There were a pile of bugs there I fixed.

Reply:
It was already up to date, found the issue, I was adding only after lastSurveyExit, not after lastExitTurnaround. This fixed it:
```
            if (lastExitTurnaround) {
                _appendCameraMountPitchItem(items, missionItemParent, seqNum, 0);
            }
```